CONTRIBUTING.md

```diff
@@ -65,16 +29,127 @@ forth here.
 Failure to adhere to these style guidelines will slow down the review process
 and block approval/merging.
 
-### Prettier
-
-This repo uses prettier to help maintain the quality and consistency of the
-content within. There is a master
-[prettier configuration file](https://github.com/solana-foundation/developer-content/blob/main/.prettierrc)
-(`.prettierrc`) in the root of this repo.
+### Prose
+
+The guidelines below are consistent with Solana Foundation Style (if you work at
+Solana Foundation, see Notion), and
```

Comment:
Seeing as this is outward-facing style guide, best to not reference any internal documentation.
```suggestion
The guidelines below are consistent with Solana Foundation Style Guide, and
```